doc/man1/qstat.1B

```diff
@@ -698,17 +698,17 @@ Show size in gigabytes.  Alternate format is used.
 Show size in megawords.  A word is considered to be 8 bytes.  
 Alternate format is used.
 
-.IP "-1" 10
+.IP "-w" 10
 For server, queue, or job long format: when used with the 
 .I -f
 option, each long format attribute is printed on one unbroken line.
 
-For job alternate format: when used with the 
-.I -n
-and/or 
-.I -s
-options, 
-reformats job's alternate qstat output to a single line.  
+.IP "-D <delimiter>" 10
+Specifies delimiter for dsv format output.  Default delimiter is a 
+vertical bar ("|").
+
+.IP "-F <json|dsv>" 10
+Specifies output format.  Format can be json or dsv.
```

Comment:
@nithinj  since -1 is till a valid suboption with "-n/-s" please keep the text that was there previously.